src/Raven.Server/Utils/TypeConverter.cs

```diff
@@ -391,6 +392,9 @@ public static dynamic ToDynamicType(object value)
             if (value is DynamicNullObject)
                 return value;
 
+            if (value is DynamicObject)
```

Comment:
Maybe `IDynamicMetaObject` ?

Reply:
Yes. Changed to `IDynamicMetaObjectProvider`